modules/exploits/windows/fileformat/publishit_pui.rb

```diff
@@ -0,0 +1,83 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'Publish-It 3.6d Buffer Overflow (SEH)',
+      'Description'  => %q{
+          This module exploits a stack based buffer overflow in Publish-It when
+          processing a specially crafted .PUI file. This vulnerability could be
+          exploited by a remote attacker to execute arbitrary code on the target
+          machine by enticing a user of Publish-It to open a malicious .PUI file.
+      },
+      'License'    => MSF_LICENSE,
+      'Author'    =>
+        [
+          'Daniel Kazimirow from Core Security Exploit Writers Team',  # Original discovery
```

Comment:
We try to keep module author names to just names so they can be counted accurately.